internal/internal_workflow_testsuite.go

```diff
@@ -463,6 +470,11 @@ func (env *testWorkflowEnvironmentImpl) executeActivity(
 		},
 		ActivityType: *activityType,
 		Input:        input,
+		Header: &shared.Header{
+			Fields: map[string][]byte{
+				testHeader: []byte("test-data"),
+			},
+		},
```

Comment:
What is the purpose of this header for activity testing?

Reply:
Removed, made this configurable so user can use on tests..